src/assets/styles/header.scss

```diff
@@ -3,6 +3,15 @@
   text-align: center;
 }
 
+.header.-sticky {
```

Comment:
One issue when hiding the branding and ending up with a new header height is that if you click on the filter events button, you end up with:

![image](https://cloud.githubusercontent.com/assets/1724544/25466306/b5758b2e-2abc-11e7-9ec2-f7675c200ffc.png)

Where the height for the filters thing is set incorrectly